packages/@integration/src/cqrs/domain/Offer.command.ts

```diff
@@ -1,14 +1,9 @@
-import { createEvent, EventsUnion } from '@marblejs/core';
+import * as t from 'io-ts';
+import { event } from '@marblejs/core';
 
 export enum OfferCommandType {
   GENERATE_OFFER_DOCUMENT = 'GENERATE_OFFER_DOCUMENT',
```

Comment:
Any reason why `enum` is used here where `const enum` is used in `Offer.event`?

Reply:
No reason, probably it is a leftover